linter/allowlists/default.txt

```diff
@@ -18,6 +18,7 @@ AuthCode
 AWC
 Back end
 back end
+Backend
```

Comment:
Why did you put this here? "Backend" is already available in the German allowlist: https://github.com/contao/package-metadata/blob/main/linter/allowlists/de.txt

In English, we used to use the notation "back end".

Reply:
Thank you!